neuralmonkey/decoders/sequence_decoder.py

```diff
@@ -0,0 +1,310 @@
+"""Abstract class for autoregressive decoding.
```

Comment:
Možná bych zašel až tak daleko, že bych modul i klásu přejmenoval na autoregressive a AutoregressiveDecoder

Reply:
I second this

Reply:
tak to pls udělej.. @jlibovicky nebude proti

Reply:
DONE